M11-1 navbar

nhn190000 opened the pull request:
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]


* **What kind of change does this PR introduce?** (Bug fix, new feature...)
 Implemented navbar using reactstrap



* **What is the current behavior?What does the code do before you made the new changes?** 
Current behavior is N/A since there was no navbar. 


* **What is the new behavior?**
Application now has a basic working navbar that has a branding and two links to "home" and "swipe". The links currently does not go anywhere.


* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No breaking changes


* **Other information**:

Codubee committed:
Renamed PR template file

nhn190000 committed:
Add navbar component

Using reactstrap, I added navbar component and added that component to app.js

nhn190000 committed:
Add color scheme to navbar and change size of text

Codubee commented on frontend/src/components/Navbar.js:
```diff
@@ -0,0 +1,37 @@
+  NavLink,
+} from 'reactstrap';
+
+const MyNavBar = (props) => {
```
Great work! Can you change this from a constant to a functional component. [Here](https://imgur.com/CO0B4U6) is an example of how to do so.

nhn190000 replied:
I'll take a look at it tonight after class!

Codubee reviewed (changes requested):
Great work! I left one comment for discussion.

nhn190000 committed:
Change component into a function and refactor code.

I changed NavBar component into a function from its original constant function and changed the code around to clean it up.

Codubee committed:
Removed .js from file name

Codubee reviewed (approved):
Great work Nam!
